repo/packages/M/marathon/0/config.json

```diff
@@ -1,147 +1,181 @@
+          "maximum": 65535,
+          "minimum": 0,
+          "type": "integer"
+        },
+        "marathon-store-timeout": {
+          "description": "Maximum time in milliseconds, to wait for persistent storage operations to complete.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "mem": {
+          "default": 512.0,
+          "description": "Memory (MB) to allocate to each Marathon instance.",
+          "minimum": 0.0,
+          "type": "number"
+        },
+        "mesos-authentication-principal": {
+          "description": "The Mesos principal used for authentication.",
+          "type": "string"
+        },
+        "mesos-authentication-secret-file": {
+          "description": "The path to the Mesos secret file containing the authentication secret.",
+          "type": "string"
+        },
+        "mesos-role": {
+          "description": "Mesos role for this framework.",
+          "type": "string"
+        },
+        "reconciliation-initial-delay": {
+          "description": "The delay, in milliseconds, before Marathon begins to periodically perform task reconciliation operations.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "reconciliation-interval": {
+          "description": "The period, in milliseconds, between task reconciliation operations.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "ssl-keystore-password": {
+          "description": "Password for the keystore supplied with the marathon/ssl-keystore-path option.",
+          "type": "string"
+        },
+        "ssl-keystore-path": {
+          "description": "Path to the SSL keystore. SSL will be enabled if this option is supplied. The keystore can be downloaded into Marathon's working directory by using the uris property.",
+          "type": "string"
+        },
+        "uris": {
+          "default": [],
+          "description": "List of URIs that will be download and made available in the current working directory of Marathon. For example this can be used to download Java keystore file for SSL configuration.",
+          "items": {
+            "pattern": "^[\\s]+",
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "zk": {
+          "default": "zk://master.mesos:2181/universe",
+          "description": "Parent ZooKeeper URL for storing state. The framework name is joined to the path of this value. Format: zk://host1:port1,host2:port2,.../path",
+          "type": "string"
+        },
+        "zk-max-versions": {
+          "description": "Limit the number of versions stored for one entity.",
+          "minimum": 0,
+          "type": "integer"
+        },
+        "zk-timeout": {
+          "description": "Timeout for ZooKeeper in milliseconds.",
+          "minimum": 0,
+          "type": "integer"
+        }
+      },
+      "required": [
+        "zk",
+        "framework-name"
```

Comment:
These two properties already have defaults.  We should be consistent with how we mark properties required.  Two options:

1. Remove these properties from the required list to match the others.
2. Add the properties that we expand without conditional guards in `marathon.json` to the required list (my preference).